zerver/views/video_calls.py

```diff
@@ -208,29 +201,34 @@ def get_bigbluebutton_url(request: HttpRequest, user_profile: UserProfile) -> Ht
 @never_cache
 @has_request_variables
 def join_bigbluebutton(
-    request: HttpRequest,
-    meeting_id: str = REQ(),
-    password: str = REQ(),
-    checksum: str = REQ(),
+    request: HttpRequest, meeting_id: str = REQ(), name: str = REQ(), password: str = REQ()
 ) -> HttpResponse:
     assert request.user.is_authenticated
 
     if settings.BIG_BLUE_BUTTON_URL is None or settings.BIG_BLUE_BUTTON_SECRET is None:
         raise JsonableError(_("BigBlueButton is not configured."))
     else:
+        signer = Signer()
+        meeting_id = signer.unsign(meeting_id)
+        name = signer.unsign(name)
+        password = signer.unsign(password)
+        create_params = urlencode(  # type: ignore[type-var] # https://github.com/python/typeshed/issues/4234
+            {
+                "meetingID": meeting_id,
+                "name": name,
+                "moderatorPW": password,
+                "attendeePW": password[:10],
```

Comment:
Can you add a comment documenting why the `attendeePw` should be related to the `moderatorPW`?

Reply:
To be honest:
It does not have to, but we need to  pass one and this was probably the easiest way to get one, while keeping the possibility that we could use it later. (Without passing it through the link)

I will add that as a comment